cvat/apps/dataset_manager/formats/cvat.py

```diff
@@ -432,7 +432,7 @@ def load(file_object, annotations):
                 )
             elif el.tag == 'image':
                 image_is_opened = True
-                frame_id = int(el.attrib['id'])
+                frame_id = annotations.match_frame(el.attrib['name'])
```

Comment:
I think you should use https://github.com/opencv/cvat/blob/develop/cvat/apps/dataset_manager/bindings.py#L571 here, which will fall back to ids and other things in the case image name did not match. This can happen, for example, in the case when  the task was created from files on a shared storage, as image names in the task will include their absolute paths, in which case the new variant doesn't work. This problem would require analysis of the whole input file, but ids can simplify things.

Another simple variant - use the name, otherwise use the id and check if basenames match, if there is one provided.

Reply:
Indeed, I thought match_frame already matched on the basename and not the full path but not sure, and I didn't follow the updates of the importer.

Reply:
You can find an example of a `DataseItem` for these purposes here: https://github.com/opencv/cvat/blob/develop/cvat/apps/dataset_manager/formats/yolo.py#L40

I think, an approach in the last line above will just fit.

Reply:
Hi, any updates?

Reply:
Done